PWGCF/TwoParticleCorrelations/DataModel/TwoParticleCorrelationsSkimmed.h

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/ASoAHelpers.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Common/DataModel/Centrality.h"
+#include "PWGCF/TwoParticleCorrelations/Core/SkimmingConfigurableCuts.h"
+#include "PWGCF/TwoParticleCorrelations/Core/EventSelectionFilterAndAnalysis.h"
+#include "PWGCF/TwoParticleCorrelations/Core/TrackSelectionFilterAndAnalysis.h"
+#include "PWGCF/TwoParticleCorrelations/Core/PIDSelectionFilterAndAnalysis.h"
+#include "Framework/runDataProcessing.h"
+
+/* TODO: this will go to its proper header at some point */
+#ifndef O2_ANALYSIS_TWOPSKIMMED_H
+#define O2_ANALYSIS_TWOPSKIMMED_H
+
+#include "Framework/ASoA.h"
+#include "Framework/AnalysisDataModel.h"
+
+namespace o2
+{
+namespace aod
+{
+/* we have to change from int to bool when bool columns work properly */
+namespace twopskim
+{
+DECLARE_SOA_COLUMN(TwoPSkimmedCollisionCentMult, centmult, std::vector<float>); //! The centrality/multiplicity pecentile
```

Comment:
Can we already use an array column as input for the event mixing? @saganatt ?

Reply:
Simplest way: can you describe in a lambda the logic of obtaining a bin number from such an array? Then you simply use LambdaBinningPolicy for mixing.
This is one array (set of values) per one collision, right? I don't have any universal logic for such case implemented.

Reply:
For a giving mixing one would use always the first entry of the array (or the second). So the lambda would probably do. WE will try it. Thanks!

Reply:
Great!
If later we will see there are more use cases and lambda code copies, and there will be such a need, I can try implementing a dedicated binning policy. But in a more distant future ;-)

Reply:
You will always know to which index in the `centmult` column the current data refers to
The skimming framework will provide it